### ISSUE TITLE
Add Jupyterlab extension

### ISSUE THREAD

**Issue by djangoliv** [lgpage/nbtutor#45]
I use jupyterlab for python trainings and your extension can be very useful for this job. 

Is there a plan to support Jupyterlab? 

Regards

**Comment by cpp-review-dune-student** [comment]
I agree , please.

**Comment by carlosal1015** [comment]
I agree please, some dependencies are broken for notebook, support for jupyterlab will be awesome.

**Comment by rootsmusic** [comment]
@djangoliv Try [jupyterlab-pytutor](https://pypi.org/project/jupyterlab-pytutor/)
